### PR TITLE
Add OpenTelemetry Instrumentation

### PR DESCRIPTION
**Pull request by yashvardhannanavati**
This PR is supposed to be addition to the changes in #508. Please only review the commit added by me. Thanks!

cc @raks-tt

### CODE REVIEW

**Comment by lipoja** [comment]
LGTM

File: README.md [hunk 1]
```diff
@@ -208,6 +208,8 @@ The custom configuration options for the REST API are listed below:
 * `IIB_LOG_LEVEL` - the Python log level of the REST API (Flask). This defaults to `INFO`.
 * `IIB_MAX_PER_PAGE` - the maximum number of build requests that can be shown on a single page.
   This defaults to `20`.
+* `IIB_OTEL_TRACING` - the boolean value which indicates if OpenTelemetry tacing is enabled in IIB.
```
Comment (chandwanitulsi) [inline README.md:211]:
```suggestion
* `IIB_OTEL_TRACING` - the boolean value which indicates if OpenTelemetry *tracing* is enabled in IIB.
```

File: README.md [hunk 2]
```diff
@@ -395,6 +397,8 @@ The custom configuration options for the Celery workers are listed below:
     }
   ```
 
+* `iib_otel_tracing` - the boolean value which indicates if OpenTelemetry tacing is enabled in IIB.
```
Comment (chandwanitulsi) [inline README.md:400]: **tracing**

**Comment by chandwanitulsi** [comment]
@yashvardhannanavati your changes look good :+1: 
we will need to merge the other PR first to merge this

**Comment by yashvardhannanavati** [comment]
Closing in favor of #522

**Pull request closed without merging**